common/jenkins-agents/nodejs10-angular/docker/Dockerfile.rhel7

```diff
@@ -15,7 +15,7 @@ LABEL com.redhat.component="jenkins-agent-nodejs-rhel7-docker" \
 ARG nexusUrl
 ARG nexusAuth
 
-ENV NODEJS_VERSION=8.11 \
```

Comment:
This is not actually used, but I thought it makes sense to "fix" it. The variable is used again in the new NodeJS 12 image so I left it here.